cterasdk/edge/nfs.py

```diff
@@ -1,21 +1,53 @@
 import logging
 
 from .enum import Mode
+from ..exception import CTERAException
 from .base_command import BaseCommand
 
 
 class NFS(BaseCommand):
     """ Gateway NFS configuration """
 
-    def is_disabled(self):
-        """ Check if the NFS server is disabled """
-        return self._gateway.get('/config/fileservices/nfs/mode') == Mode.Disabled
+    def get_configuration(self):
```

Comment:
Same comment as the one made re `get_status`. We should rename the rest of the modules to use this standard. Have all of the modules expose `get_configuration()` and `get_status()` or `config()` and `status()`

Reply:
Addressed